src/style-banner.scss

```diff
@@ -169,6 +168,70 @@
   }
 }
 
+// =-=-=-=-= iconic -=-=-=-=-=-=
+
+:host([iconic]) {
+  @extend %auro-banner;
+
+  $iconBgSize: 120px;
+  .iconAbsolute {
+    position: absolute;
+  }
+  .iconContainer {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    position: relative;
+    top: -$iconBgSize/6;
+  }
+
+  .roundIconBg {
+    width: $iconBgSize;
+    height: $iconBgSize;
+    background: #c4c4c4;
+    border-radius: 50%;
+    padding: 20px;
```

Comment:
This value is weird to me. This is not a token, so that is not standard with the grid that has been established?